pkgs/development/tools/kind/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, lib, buildGoPackage, fetchFromGitHub }:
+
+with stdenv.lib;
+
+buildGoPackage rec {
+  name = "kind-${version}";
+  version = "20181003-${stdenv.lib.strings.substring 0 7 rev}";
```

Comment:
So it is 20181003 = 2018 October 3, the day of referenced commit

Reply:
Hmm, not sure to understand:/
In fact, I was talking about missing dashes: `s/20181003/2018-10-03`.

Also, I think the date is more the date of the day you are writing the package since the goal is to have an ordered package name. But in this PR, `10-03` is fine.

Reply:
Ok i understand, i fixed it.